modules/distribution/src/repository/jaggeryapps/cdm/modules/sso.js

```diff
@@ -71,12 +77,15 @@ var ssoMod = require("sso");
  		sso.relayState = (referer ? referer : sso.relayState);
  		sso.relayState = sso.relayState + request.getQueryString(); // append query string
  		sso.encodedSAMLLogoutRequest = ssoMod.client.getEncodedSAMLLogoutRequest(user, sso.sessionIndex , sso.issuer);
- 		sso.log.debug("Logout request recieved from session id ###: " + sso.sessionId );
- 		
  		var postUrl = sso.address + sso.ssoService;
- 		sso.log.debug(sso.sessionId);
+
+		if(log.isDebugEnabled()){
+			sso.log.debug("Logout request recieved from session id ###: " + sso.sessionId );
```

Comment:
Here, in this particular line, you seem to have used "###:" to separate out the value logged against the string content. Similarly, I've noticed instances where you've used ":::" and sometimes just ":" too for the same purpose. If possible, try to stick to just one approach (My personal preference is to go with the third option) so the code would look more consistent and elegant.